src/sfctl/tests/clustermanifest1.txt

```diff
@@ -0,0 +1 @@
>\r\n    <Section Name=\"Management\">\r\n      <Parameter Name=\"ImageStoreConnectionString\" Value=\"fabric:ImageStore\" \/>\r\n    <\/Section>\r\n    <Section Name=\"NamingService\">\r\n      <Parameter Name=\"MinReplicaSetSize\" Value=\"1\" \/>\r\n      <Parameter Name=\"PlacementConstraints\" Value=\"NodeTypeName==NodeType0\" \/>\r\n      <Parameter Name=\"TargetReplicaSetSize\" Value=\"1\" \/>\r\n    <\/Section>\r\n    <Section Name=\"PlacementAndLoadBalancing\">\r\n      <Parameter Name=\"QuorumBasedReplicaDistributionPerUpgradeDomains\" Value=\"true\" \/>\r\n    <\/Section>\r\n    <Section Name=\"ReconfigurationAgent\">\r\n      <Parameter Name=\"IsDeactivationInfoEnabled\" Value=\"true\" \/>\r\n    <\/Section>\r\n    <Section Name=\"Security\">\r\n      <Parameter Name=\"ClientRoleEnabled\" Value=\"true\" \/>\r\n      <Parameter Name=\"ClusterCredentialType\" Value=\"None\" \/>\r\n      <Parameter Name=\"DisableFirewallRuleForDomainProfile\" Value=\"false\" \/>\r\n      <Parameter Name=\"DisableFirewallRuleForPrivateProfile\" Value=\"false\" \/>\r\n      <Parameter Name=\"DisableFirewallRuleForPublicProfile\" Value=\"false\" \/>\r\n      <Parameter Name=\"ServerAuthCredentialType\" Value=\"None\" \/>\r\n    <\/Section>\r\n    <Section Name=\"Setup\">\r\n      <Parameter Name=\"FabricDataRoot\" Value=\"C:\\ProgramData\\SF\" \/>\r\n      <Parameter Name=\"FabricLogRoot\" Value=\"C:\\ProgramData\\SF\\Log\" \/>\r\n    <\/Section>\r\n    <Section Name=\"Trace\/Etw\">\r\n      <Parameter Name=\"Level\" Value=\"4\" \/>\r\n    <\/Section>\r\n    <Section Name=\"UpgradeOrchestrationService\">\r\n      <Parameter Name=\"AutoupgradeEnabled\" Value=\"True\" \/>\r\n      <Parameter Name=\"ClusterId\" Value=\"d4689c9a-e231-447f-8bfb-5a321a13d47d\" \/>\r\n      <Parameter Name=\"MinReplicaSetSize\" Value=\"1\" \/>\r\n      <Parameter Name=\"PlacementConstraints\" Value=\"NodeTypeName==NodeType0\" \/>\r\n      <Parameter Name=\"TargetReplicaSetSize\" Value=\"1\" \/>\r\n    <\/Section>\r\n    <Section Name=\"WinFabCrashDump\">\r\n      <Parameter Name=\"DataDeletionAgeInDays\" Value=\"7\" \/>\r\n      <Parameter Name=\"FolderType\" Value=\"WindowsFabricCrashDumps\" \/>\r\n      <Parameter Name=\"IsEnabled\" Value=\"true\" \/>\r\n      <Parameter Name=\"ProducerType\" Value=\"FolderProducer\" \/>\r\n    <\/Section>\r\n    <Section Name=\"WinFabEtlFile\">\r\n      <Parameter Name=\"DataDeletionAgeInDays\" Value=\"7\" \/>\r\n      <Parameter Name=\"IsEnabled\" Value=\"true\" \/>\r\n      <Parameter Name=\"ProducerType\" Value=\"EtlFileProducer\" \/>\r\n    <\/Section>\r\n    <Section Name=\"WinFabPerfCtrFolder\">\r\n      <Parameter Name=\"DataDeletionAgeInDays\" Value=\"7\" \/>\r\n      <Parameter Name=\"FolderType\" Value=\"WindowsFabricPerformanceCounters\" \/>\r\n      <Parameter Name=\"IsEnabled\" Value=\"true\" \/>\r\n      <Parameter Name=\"ProducerType\" Value=\"FolderProducer\" \/>\r\n    <\/Section>\r\n  <\/FabricSettings>\r\n<\/ClusterManifest>"}
```

Comment:
You shouldn't check in files like this for the sake of file keeping. Let's talk about how you can test this without having to check in an entire cluster manifest.